features/step_definitions/basic_step.rb

```diff
@@ -0,0 +1,39 @@
+Then(/^show me the page$/) do
+  save_and_open_page
+end
+
+Given(/^the following user exist$/) do |table|
+  table.hashes.each do | user |
+    User.create(user)
```

Comment:
Step definition is removed. New step definition is added to branch for login test.